transport/nio/src/main/java/org/kaazing/gateway/transport/nio/NioSystemProperty.java

```diff
@@ -52,7 +52,11 @@
     // Note, the idle usage is using the mina netty idle timeout which may be set
     // by higher layers. Logic for this is in NioIdleFilter
     TCP_IDLE_TIMEOUT("org.kaazing.gateway.server.transport.tcp.IDLE_TIMEOUT", Integer.toString(0)),
-    UDP_IDLE_TIMEOUT("org.kaazing.gateway.server.transport.udp.IDLE_TIMEOUT", "60");
+    UDP_IDLE_TIMEOUT("org.kaazing.gateway.server.transport.udp.IDLE_TIMEOUT", "60"),
+    UDP_CLIENT_RECEIVE_BUFFER_SIZE("org.kaazing.gateway.server.transport.udp.CLIENT_RECEIVE_BUFFER_SIZE", "65536"),
+    UDP_CLIENT_SEND_BUFFER_SIZE("org.kaazing.gateway.server.transport.udp.CLIENT_SEND_BUFFER_SIZE", "65536"),
+    UDP_SERVER_RECEIVE_BUFFER_SIZE("org.kaazing.gateway.server.transport.udp.SERVER_RECEIVE_BUFFER_SIZE", "2097152"),
+    UDP_SERVER_SEND_BUFFER_SIZE("org.kaazing.gateway.server.transport.udp.SERVER_SEND_BUFFER_SIZE", "2097152");
```

Comment:
Are these server values big enough for say 1000s of clients? If not we'll need to increase these defaults or make the property settable in a documented and supported way and document customers may need to set it.